android/assets/jsons/translations/Traditional_Chinese.properties

```
@@ -281,45 +281,45 @@ When Friends: =
  # Requires translation!
 When Allies: = 
  # Requires translation!
-The unique luxury is one of: = 
+The unique luxury is one of: = 獨特的奢侈資源是以下之一：
 Demand Tribute = 要求貢品
  # Requires translation!
 Tribute Willingness = 
  # Requires translation!
 At least 0 to take gold, at least 30 and size 4 city for worker = 
  # Requires translation!
-Take [amount] gold (-15 Influence) = 
+Take [amount] gold (-15 Influence) = 索取[數量]金幣(-15 影響力)
```

In case you don’t know what’s him talking about, it should be:
```suggestion
Take [amount] gold (-15 Influence) = 索取[amount]金幣(-15 影響力)
```

---

I understand, thanks for your addition.